README.md

```diff
@@ -16,43 +16,50 @@ Installation requires the following steps:
 1. Set up the required Active Directory security permissions for the application to authorize it to read threat intelligence data and activity reports for your organization.
 1. Create an Access Key that will allow the application to connect to the Alert Logic Cloud Defender and Cloud Insight backend.
 1. Download and deploy a custom ARM template to Microsoft Azure to create functions for collecting and managing O365 log data
-1. Verify that installation was successful using Alertlogic CloudDefender UI.
+1. Verify that installation was successful using the Alertlogic UI under Configuration -> Deployments -> All Deployments -> Log Sources -> Filter by "Office 365" collection method.
 
```

Comment:
The collection method is actually Push (Office 365, CloudWatch) - maybe it's worth using the full name? Though, perhaps, it's better to filter by "Office65 logs" Source Type?

Reply:
Agree and it has been updated.